app/src/main/kotlin/nl/q42/template/logging/CrashlyticsAntilog.kt

```diff
@@ -0,0 +1,40 @@
+package nl.q42.template.logging
+
+import com.google.firebase.crashlytics.ktx.crashlytics
+import com.google.firebase.ktx.Firebase
+import io.github.aakira.napier.Antilog
+import io.github.aakira.napier.DebugAntilog
+import io.github.aakira.napier.LogLevel
+
+/** A value suitable for Crashlytics */
+private const val MAX_CHARS_IN_LOG = 1200
+
+class CrashlyticsAntilog : Antilog() {
+
+    private val logcatAntilog = DebugAntilog()
+
+    override fun performLog(
+        priority: LogLevel,
+        tag: String?,
+        throwable: Throwable?,
+        message: String?
+    ) {
+        // also send to logcat
+        logcatAntilog.log(priority, tag, throwable, message)
+
+        val limitedMessage = message?.take(MAX_CHARS_IN_LOG) // to avoid OutOfMemoryError's
+
+
+        if ((priority == LogLevel.INFO || priority == LogLevel.DEBUG || priority == LogLevel.WARNING) && limitedMessage != null) {
```

Comment:
`limitedMessage != null`. Does this mean that debug/info/warn logging a throwable without a message does nothing / is ignored? That's not what I would expect when message is nullable: I would expect the throwable to be logged.

I would also make this a when clause (or if else). Instead of 2 ifs, that should never trigger both.